flow/project.py

```diff
@@ -4184,6 +4231,12 @@ def foo(job):
         # TODO: Add aggregator argument and pass through to FlowGroupEntry.
         if name in cls._GROUP_NAMES:
             raise ValueError(f"Repeat definition of group with name '{name}'.")
+        if any(
+            name == operation_name for operation_name, _ in cls._OPERATION_FUNCTIONS
+        ):
+            raise ValueError(
+                f"Cannot create user group with name of existing operation {name}"
```

Comment:
"User group" makes this sound like a UNIX authentication problem...
```suggestion
                f"Cannot create a group with the same name as the existing operation {name}"
```